### PR TITLE
Sonar-fixes for String formatting

### PR DESCRIPTION
**Pull request by mazarag2**
#### What does this PR do?

Address sonar issues from https://sonarcloud.io/project/issues?id=ddf&resolved=false&rules=java%3AS3457&severities=MAJOR

#### Ask 2 committers to review/merge the PR and tag them here.

@pklinef
@shaundmorris

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-branding-plugin/src/main/java/org/codice/ddf/branding/impl/DdfBrandingPlugin.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public void init() {
       vendorImage = propertiesConfiguration.getString("branding.vendor.image", "/ddf/logo.png");
       favIcon = propertiesConfiguration.getString("branding.favicon", "/ddf/favicon.png");
     } catch (ConfigurationException e) {
-      LOGGER.info("Unable to read properties file {}", brandingPropertiesFilePath, e.getMessage());
+      LOGGER.info("Unable to read properties file properties {} Message {}", brandingPropertiesFilePath, e.getMessage());
```
Comment (mazarag2) [inline distribution/ddf-branding-plugin/src/main/java/org/codice/ddf/branding/impl/DdfBrandingPlugin.java:63]: Decided to add more to the message here around the parameters

**Review by zta6 — APPROVED** [review]

**Review by pklinef — APPROVED** [review]

**Comment by mazarag2** [comment]
build now

**Review by shaundmorris — APPROVED** [review]

**Comment by mazarag2** [comment]
build now

**Comment by mazarag2** [comment]
build now

**Comment by mazarag2** [comment]
build now

**Pull request merged**